pouchdb-core/pouchdb-core-tests.ts

```diff
@@ -129,4 +129,53 @@ namespace PouchDBCoreTests {
       // Callback version with docId and rev
       db.remove(id, rev, {}, (res: PouchDB.Core.Response) => {});
     }
+
+    function testChanges() {
+        type MyModel = { foo: string };
+        const db = new PouchDB<MyModel>();
+
+        db.changes({
```

Comment:
This is how I think test should be made:
Test all possible combinations of function options (see second call of `db.change`).
Also see how I check all properties returned in on change event.
This way I cannot make mistake by unintentionally remove property or change its type.